proto/cv/scanner/v1/scanner.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package cv.scanner.v1;
+
+import "google/api/annotations.proto";
+import "google/api/http.proto";
+import "ssn/annotator/v1/annotator.proto";
+import "ssn/type/candidate.proto";
+
+option go_package = "github.com/e-conomic/vmlapis/gen/go/cv/cv/v1;cv";
```

Comment:
One too many `cv`s?

Reply:
Nice catch btw. Our compiler **does not care about this definition**. Seems like it resolves it based on filepath like 
"github.com/e-conomic/vmlapis/gen/go/cv/scanner/v1"